packages/react/src/primitives/Button/Button.tsx

```diff
@@ -37,7 +38,11 @@ const ButtonInner: PrimitiveWithForwardRef<ButtonProps, 'button'> = (
       type={type}
       {...rest}
     >
-      {isLoading && loadingText ? <span>{loadingText}</span> : children}
+      {isLoading && loadingText ? (
+        <Text as="span">{loadingText}</Text>
```

Comment:
Hehe, `Text` didn't even exist when this was written.

Reply:
I was wondering if this `span` is really required. Looks like not styles applied on it